tests/Smoke/InstallViaComposerTest.php

```diff
@@ -75,6 +75,9 @@ public function testInstallationViaPathIsPossible()
                 [
                     'type' => 'path',
                     'url' => __DIR__.'/../..',
+                    'options' => [
+                        'symlink' => false,
```

Comment:
any reasoning ?

Reply:
This test would fail - without it packages was installed via symlink and then calling `php-cs-fixer` would require `autoload.php` from project's root `vendor`, not the one created for the test.

Reply:
why it's not failing on master branch when symlink is not there?

Reply:
it uses original code, so all is good - with these changes there is no `vendor/autoload.php` as all was moved to `dev-tools`